modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -66,7 +66,7 @@ HTTP requests sent to Runtime Fabric on VMs / Bare Metal are ignored.
 
 * *Accept HTTP requests only*
 +
-Because certificates are not used in this case, click *Add Domain* and manually enter domains for routing requests to applications. Skip the xref:configure-a-tls-context-for-hh[Configure a TLS context for HTTPS] task.
+Because certificates are not used in this case, click *Add Domain* and manually enter domains for routing requests to applications. Skip the xref:.adoc[Configure a TLS context for HTTPS] task.
```

Comment:
```suggestion
Because certificates are not used in this case, click *Add Domain* and manually enter domains for routing requests to applications. Skip the xref:configure-a-tls-context-for-hh.adoc[Configure a TLS context for HTTPS] task.
```